### PR TITLE
Update CNCF security white-paper link

### PR DESCRIPTION
**Pull request by PushkarJ**
Minor change to refer to kubernetes blog post for cncf security whitepaper

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm
/approve
/sig security
/kind cleanup

File: content/en/blog/_posts/2021-04-15-Three-Tenancy-Models-for-Kubernetes.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The [Virtual Cluster](https://github.com/kubernetes-sigs/multi-tenancy/tree/mast
 
 ## Security considerations
 
-Cloud native security involves different system layers and lifecycle phases as described in the [Cloud Native Security Whitepaper](https://www.cncf.io/blog/2020/11/18/announcing-the-cloud-native-security-white-paper/) from CNCF SIG Security.  Without proper security measures implemented across all layers and phases, Kubernetes tenant isolation can be compromised and a security breach with one tenant can threaten other tenants.
+Cloud native security involves different system layers and lifecycle phases as described in the [Cloud Native Security Whitepaper](https://kubernetes.io/blog/2020/11/18/cloud-native-security-for-your-clusters) from CNCF SIG Security.  Without proper security measures implemented across all layers and phases, Kubernetes tenant isolation can be compromised and a security breach with one tenant can threaten other tenants.
```
Comment (sftim) [inline content/en/blog/_posts/2021-04-15-Three-Tenancy-Models-for-Kubernetes.md:47]: nit: As this is now an internal link, we'd normally put:
```suggestion
Cloud native security involves different system layers and lifecycle phases as described in the [Cloud Native Security Whitepaper](/blog/2020/11/18/cloud-native-security-for-your-clusters) from CNCF SIG Security.  Without proper security measures implemented across all layers and phases, Kubernetes tenant isolation can be compromised and a security breach with one tenant can threaten other tenants.
```

**Comment by PushkarJ** [comment]
@sftim added a commit with your suggestion

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**